plugins/catalog-import/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "@backstage/plugin-catalog-import",
+  "version": "0.1.1-alpha.26",
```

Comment:
Remember to bump versions here :grin:

Reply:
Bumped to 0.2.0 along with all backstage dependencies. I have a problem merging this branch to master though...